packages/cookies/test/response-cookies.test.ts

```diff
@@ -64,6 +64,26 @@ test('reflect .set into `set-cookie`', async () => {
   )
 })
 
+it('reflect .set all options attributes into `set-cookie`', async () => {
+  const headers = new Headers()
+  const cookies = new ResponseCookies(headers)
+  const options = {
+    domain: 'custom-domain',
+    path: 'custom-path',
+    secure: true,
+    sameSite: 'strict' as 'strict' | 'lax' | 'none',
+    expires: new Date(2100, 0, 1, 12, 0, 0),
+    httpOnly: true,
+    maxAge: 0,
+    priority: 'high' as 'low' | 'medium' | 'high',
+  }
+  cookies.set('first-name', 'first-value', options)
```

Comment:
```suggestion
  cookies.set('first-name', 'first-value', {
    domain: 'custom-domain',
    path: 'custom-path',
    secure: true,
    sameSite: 'strict',
    expires: new Date(2100, 0, 1, 12, 0, 0),
    httpOnly: true,
    maxAge: 0,
    priority: 'high',
  })
```

This way you don't need to cast types.

Reply:
Okay, nice